vng_api_common/authorizations/serializers.py

```diff
@@ -54,8 +54,8 @@ def validate(self, attrs):
             autorisaties_obj = self.instance.autorisaties.all()
             heeft_alle_autorisaties_obj = self.instance.heeft_alle_autorisaties
 
-        autorisaties = validated_attrs.get('autorisaties', None) or autorisaties_obj
-        heeft_alle_autorisaties = validated_attrs.get('heeft_alle_autorisaties', None) or heeft_alle_autorisaties_obj
+        autorisaties = validated_attrs.get('autorisaties', autorisaties_obj)
+        heeft_alle_autorisaties = validated_attrs.get('heeft_alle_autorisaties', heeft_alle_autorisaties_obj)
```

Comment:
same here with the default being moved - what if I include the key but set it to `null`?